examples/oauth2-codeVerifier.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {OAuth2Client} = require('../build/src/index');
+const http = require('http');
+const url = require('url');
+const querystring = require('querystring');
+const opn = require('opn');
+
+// Download your OAuth2 configuration from the Google
+const keys = require('./oauth2.keys.json');
+
+/**
+ * Start by acquiring a pre-authenticated oAuth2 client.
+ */
+async function main() {
+  try {
+    const oAuth2Client = await getAuthenticatedClient();
+    // Make a simple request to the Google Plus API using our pre-authenticated client. The `request()` method
+    // takes an AxiosRequestConfig object.  Visit https://github.com/axios/axios#request-config.
+    const url = 'https://www.googleapis.com/plus/v1/people?query=pizza';
+    const res = await oAuth2Client.request({url})
+    console.log(res.data);
+  } catch (e) {
+    console.error(e);
```

Comment:
Use `main().catch(console.error)` instead of a try block here.

Reply:
fixed!